src/main/java/com/shyb/evcharging/user/api/UserController.java

```diff
@@ -0,0 +1,81 @@
+package com.shyb.evcharging.user.api;
+
+import com.shyb.evcharging.user.application.UserService;
+import com.shyb.evcharging.user.domain.User;
+import com.shyb.evcharging.user.dto.EmailDuplicateCheckRequestDto;
+import com.shyb.evcharging.user.dto.UserModifyRequestDto;
+import com.shyb.evcharging.user.dto.UserRequestDto;
+import com.shyb.evcharging.user.dto.UserResponseDto;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * 사용자 생성, 조회, 수정, 삭제에 대한 요청 및 응답을 처리합니다.
+ */
+
+@RestController
+@RequestMapping("/users")
```

Comment:
API가 변하는 부분(버저닝)과 외부 API, 내부 API 구분을 위해 아래와 같이 사용하는 경우도 있습니다. 참고 부탁드려요.

```suggestion
@RequestMapping("/api/v1/users")
```

내부는 `webapi`, 외부는 `api` 요런식으로 분리해서 사용할수도 있어요.
그리고 인증이 적용될 경로와 적용되지 않을 경로(헬스체크 등)을 구분할 필요가 있는 경우가 있어서 앞에 위와 같이 접두사가 붙는게 좋을 것 같아요.